glomap/estimators/relpose_estimation.cc

```diff
@@ -20,14 +20,15 @@ void EstimateRelativePoses(ViewGraph& view_graph,
 
   const int64_t num_image_pairs = valid_pair_ids.size();
   const int64_t kNumChunks = 10;
-  const int64_t inverval = std::ceil(num_image_pairs / kNumChunks);
+  const int64_t interval =
+      std::ceil(static_cast<double>(num_image_pairs) / kNumChunks);
```

Comment:
```suggestion
      std::min(num_image_pairs / kNumChunks, 1);
```

Reply:
nit: this would be a safer alternative that handles the full range of int64 numbers while double may not hold the full range of int64 numbers. This is unrealistic though, because this for loop would probably run for a very very long time.

Reply:
Thanks so much! I really did not take this problem into account. But as you say, I think there won't be anyone processing so many pictures. 🤣